media/js/firefox/whatsnew/whatsnew-61.js

```diff
@@ -0,0 +1,162 @@
+    Display the FxA iframe
+
+2. User is logged in to Sync, but does not have any mobile devices set up:
+    Display App/Play store badges for Firefox Mobile
+
+    i. Send to Device widget is available in the user's locale:
+        Display the Send to Device widget, configured for Firefox Mobile
+    ii. Send to Device widget is *not* available in the user's locale:
+        Display a QR code for Firefox Mobile
+
+3. User is logged in to Sync, and *does* have a mobile device set up:
+
+    i. If they're in Germany, Austria, or Switzerland:
+        a. Send to Device widget is available in the user's locale:
+            Display the Send to Device widget, configured for Klar
+        b. Send to Device widget is *not* available in the user's locale:
+            Display a QR code for Klar
+    ii. If they're in any other country *or* geolocation fails
+        a. Send to Device widget is available in the user's locale:
+            Display the Send to Device widget, configured for Focus
+        b. Send to Device widget is *not* available in the user's locale:
+            Display a QR code for Focus
+
+*/
+
+(function(Mozilla, $) {
+    'use strict';
+
+    Mozilla.WNP61 = {
+        client: Mozilla.Client,
+        klarCountryCodes: ['at', 'ch', 'de'],
+        logoFocus: document.getElementById('logo-focus'),
+        logoFx: document.getElementById('logo-fx'),
+        mainContent: document.querySelector('.main-content'),
+        $strings: $('#strings')
+    };
+
+    Mozilla.WNP61.showFxa = function() {
+        // Show the content
+        this.mainContent.classList.add('show-fxa');
+
+        // Set the title
+        document.title = this.$strings.data('fxaccount-title');
+
+        // initialize the FxA iframe
+        this.client.getFirefoxDetails(function(data) {
+            Mozilla.FxaIframe.init({
+                distribution: data.distribution,
+                gaEventName: 'whatsnew-fxa'
+            });
+        });
+    };
+
+    Mozilla.WNP61.showFirefoxMobile = function() {
+        // Show the content
+        this.mainContent.classList.add('show-fx-mobile');
+
+        // Set the title
+        document.title = this.$strings.data('fxmobile-title');
+
+        var form = new Mozilla.SendYourself('send-firefox');
+        form.init();
+    };
+
+    Mozilla.WNP61.showFocus = function(countryCode) {
+        // Show the content
+        this.mainContent.classList.add('show-focus');
+
+        // Set the title
+        document.title = this.$strings.data('fxfocus-title');
+
+        // swap out Firefox logo for Focus logo
+        this.logoFx.classList.replace('showing', 'hiding');
+
+        setTimeout(function() {
+            this.logoFocus.classList.replace('hiding', 'showing');
+        }.bind(this), 150);
+
+        var form = new Mozilla.SendYourself('send-focus');
+        form.init(countryCode);
+    };
+
+
+    Mozilla.WNP61.showKlar = function(countryCode) {
```

Comment:
This function is almost the same as `showFocus` - could the two perhaps be combined to make things more DRY? (not a blocker)